package.json

```diff
@@ -56,7 +56,7 @@
     "unit-viewer": "yarn mocha --testMatch viewer/**/*-test.js",
     "unit-flow": "bash flow-report/test/run-flow-report-tests.sh",
     "unit": "yarn unit-flow && yarn mocha",
-    "unit:ci": "NODE_OPTIONS=--max-old-space-size=8192 npm run unit --forbid-only",
```

Comment:
oh. we need to support this in the script. can you not remove ?

Reply:
The `--forbid-only` flag didn't work like this before. This PR adds it to `run-mocha-tests.js` and gives it a default value of `true` in CI.

Reply:
What do you mean it doesn't work when used like this?

I know it doesn't work _at all_ w/o this PR or #15383 , but I don't understand you're meaning here

Reply:
Two reasons:
- Before this PR we didn't pipe the `--forbid-only` option into mocha
- We would need to do `npm run unit -- --forbid-ony` to pass the flag all the way down

Instead of messing with the flags in the CI scripts, I would prefer to just look at `process.env.CI`. We already do this for `--parallel` in `run-mocha-tests.js`.